### DISCUSSION TITLE
Get boolean options without having to specify --option False/True

### DISCUSSION THREAD

**Discussion by yesseruser** [pallets/click#2665] [Q&A] [answered]
With Cocona (a C# equivalent of Click), when I have a boolean option, I can simply use it by using --option without specifying true or false. Is it possible to get the same behaviour in Click?

**Comment by ThiefMaster** [comment]
try searching for flags in the docs..

**Reply by yesseruser** [reply, marked as answer]
> try searching for flags in the docs..

Found a [stack overflow post](https://stackoverflow.com/q/52216247/15143303) where `is_flag=true` on an `option` was used.
